modules/math/src/geospatial/ellipsoid.js

```diff
@@ -283,7 +288,7 @@ export default class Ellipsoid {
    */
   geodeticSurfaceNormal(cartesian, result = [0, 0, 0]) {
     vec3.multiply(result, cartesian, this._oneOverRadiiSquared);
-    return vec3.normalize(result, result);
+    return vec3.normalize(new Vector3(), result);
```

Comment:
Why this change? you want the result to be a `Vector3()`?

But this function is supposed to return result if provided, not a new `Vector3`

Change the default argument to `result=new Vector3()` instead of `result=[0,0,0]`?